java-checks/src/main/java/org/sonar/java/checks/helpers/QuickFixHelper.java

```diff
@@ -102,6 +113,72 @@ public static SyntaxToken previousToken(Tree tree) {
     return previousToken(parent);
   }
 
+  /**
+   * Returns the preceding variable in a mutli-variable declaration.
+   *
+   * @param current The target variable
+   * @return An Optional with the preceding variable, Optional.empty otherwise.
+   */
+  public static Optional<VariableTree> previousVariable(VariableTree current) {
+    Tree parent = current.parent();
+    if (parent.is(Tree.Kind.BLOCK, Tree.Kind.INITIALIZER, Tree.Kind.STATIC_INITIALIZER)) {
+      List<Tree> children = ((BlockTreeImpl) parent).children();
+      return previousVariable(current, children, TRIMMING_OFFSET_IN_BLOCK_CHILDREN);
+    } else if (parent.is(Tree.Kind.CLASS)) {
+      List<Tree> members = ((ClassTree) parent).members();
+      return previousVariable(current, members, TRIMMING_OFFSET_IN_CLASS_MEMBERS);
+    } else {
```

Comment:
Warning, there's also `parent.is(Tree.Kind.METHOD)` where we could probably return  no previous variable or next variable. No need to look in `((MethodTree) parent).parameters()`
Same for `parent.is(Tree.Kind.LIST)`